src/SmartComponents/Inventory/applications/vulnerabilities/Vulnerabilities.js

```diff
@@ -3,17 +3,36 @@ import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import { fetchCveListBySystem } from '../../../../redux/actions/applications';
 import VulnerabilitiesCves from './VulnerabilitiesCves';
-import { Card, CardBody } from '@patternfly/react-core';
+import { Card, CardBody, Tooltip } from '@patternfly/react-core';
+import { InfoCircleIcon } from '@patternfly/react-icons';
 import { createCveListBySystem } from './DataMapper';
 import { sortable, cellWidth } from '@patternfly/react-table';
 import './vulnerabilities.scss';
 
+const cvssBaseDescription = 'All CVEs use Common Vulnerability Scoring System v3 except where noted.';
+
 const header = [
     { title: '', key: 'impact', transforms: [ sortable, cellWidth(10) ]},
     { title: 'Name', key: 'synopsis', transforms: [ sortable, cellWidth(10) ]},
     { title: 'Description', key: 'description', transforms: [ cellWidth('max') ]},
-    { title: 'CVSS Base Score', key: 'cvss_score', transforms: [ sortable, cellWidth(10) ]},
-    { title: 'Publish date', key: 'public_date', transforms: [ sortable, cellWidth(10) ]}
+    {
+        title: (
+            <React.Fragment>
+                <Tooltip
+                    position="top"
+                    content={ <div>{ cvssBaseDescription }</div> }
+                >
+                    <span className="table-header-icon">
```

Comment:
Is there a reason to wrap this in a `span` tag instead of just using the icon?

Reply:
No, actually it's not. Fixed.